tendrl/commons/objects/__init__.py

```diff
@@ -162,6 +162,16 @@ def save(self, update=True, ttl=None):
                 NS._int.wreconnect()
                 NS._int.wclient.write(item['key'], item['value'], quorum=True)
 
+    def load_all(self):
+        value = '/'.join(self.value.split('/')[:-1])
+        etcd_resp = NS._int.client.read(value)
```

Comment:
@anmolbabu I have filed an issue https://github.com/Tendrl/commons/issues/624 , will create the wrapper and submit a PR